blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Simplifying Firmware Updates in Dasharo'
+abstract: "Updating your firmware is an important part of keeping your device
+          secure and reliable. Making this process as easy and reliable as
+          possible is, therefore, a big focus for Dasharo. In this article we'll
+          dive into the latest feature that make that process a bit easier, and
+          talk about where we want to go from here."
+cover: /covers/dasharo-sygnet.svg
+author: michal.kopec
+layout: post
+published: true
+date: 2023-09-12
+archives: "2023"
+
+tags:
+  - coreboot
+  - UEFI
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+The word *firmware* comes from the fact that it sits somewhere in between
+*soft*ware and *hard*ware. This term refers to the fact that firmware provides
+an abstraction layer for hardware, so that software (e.g. OS) can make use of it
+in a more generic manner.
+
+Firmware also sits between hardware and software in a different category, which
+is how hard it is to modify. While software is easy to modify and update,
```

Comment:
```suggestion
is "how hard it is to modify". While software is easy to modify and update,
```

It is actually hard to understand the meaning. I supposed this is what was in your mind right?

Reply:
Exactly. Suggestion applied in eb660858d2cc0c15dfd4a7cce6caee520ad9831a